website/static/js/projectorganizer.js

```diff
@@ -782,7 +783,13 @@ function _poMultiselect(event, tree) {
                     return false;
                 });
             } else {
-                createBlankProjectDetail();
+                detailTemplateContext = {
+                    itemsCount: selectedRows.length
+                };
+                var theParentNode = selectedRows[0].parent();
+                var displayHTML = multiItemDetailTemplateNoAction(detailTemplateContext);
+                $('.project-details').html(displayHTML);
+                $('.project-details').show();
```

Comment:
This jQuery object should be cached to be avoid multiple DOM traversals; this selector in particular is used throughout this module.